app/js/components/drugOrderEntry/addForm/AddForm.jsx

```diff
@@ -81,6 +83,16 @@ export class AddForm extends React.Component {
     this.setState({ formType: formType.trim() });
   }
 
+  checkIfDrugHasActiveOrder = (drug) => {
+    const { filteredOrders } = this.props.fetchOrdersReducer;
+    const drugOrders = filteredOrders.filter(order => order.type === 'drugorder');
+    const findDrug = drugOrders.filter(order => order.drug.display === drug);
```

Comment:
Please use clearer variable names.
1: Equating a `display` name to a drug looks like a `contradiction`. Use displayName or something similar instead of `drug` as a parameter.
2: Variable name `findDrug` sounds like a function and not a drug or object. Find a name that better describes a matching orders that were found. My guess it has to suit an array since you used a filter.

Reply:
Thanks for the feedback, will work on your suggestions.

Reply:
Done